tables/TaQL/TableGram.yy

```diff
@@ -1294,14 +1305,13 @@ tfnamen:   tfname {
    concatenate and the GIVING/INTO to make the concat table persistent.
 */
 tfname:    tfcommand {
-	       theFromQueryDone = True;
```

Comment:
How did you eliminate the need for `theFromQuery`? To which other change does this correspond?